src/future-hopr-lib-components/Layout/index.tsx

```diff
@@ -37,14 +37,7 @@ const Layout: React.FC<{
   children?: any;
   drawer?: boolean;
   drawerItems?: {}[];
-}> = ({
-  className = '',
-  children,
-  itemsNavbarRight,
-  tallerNavBarOnMobile,
-  drawer,
-  drawerItems,
-}) => {
+}> = ({ className = '', children, itemsNavbarRight, tallerNavBarOnMobile, drawer, drawerItems }) => {
```

Comment:
@camilovegag, please check if we can make an exception for destructuring like this one to have singleAttributePerLine